torch/testing/_internal/distributed/rpc/dist_examples_test.py

```diff
@@ -0,0 +1,30 @@
+import torch.distributed.rpc as rpc
```

Comment:
Since you've already created a new `examples` folder, we don't need a separate file here. You can merge this test class into `parameter_server.py` and rename it `paramerter_server_test.py`.

See `torch.testing._internal.distributed.rpc.jit.rpc_test` as an example on how tests from sub-folders are imported.